src/containers/singleEvent/index.tsx

```diff
@@ -29,30 +31,54 @@ const CenteredContainer = styled.div`
 
 interface SingleEventProps {
   readonly events: EventsReducerState['upcomingEvents'];
+  readonly eventAnnouncements: AnnouncementsReducerState['announcements'];
```

Comment:
I am confused. What is in the store? 
- EventsReducerState
- AnnouncementsReducerState
- EventAnnouncementsReducerState

It seems like this wants to reference EventAnnouncementReducerState, not AnnouncementReducerState. 

What if we put the eventAnnouncements in one of the existing reducers?

Reply:
You are absolutely correct - it should reference EventAnnouncementReducerState.

We cannot use an existing reducer because AnnouncementsReducerState uses Announcement, which are _non-event_ Announcements (for the announcements page).

I extended Announcement to define EventAnnouncement, with an addition of the eventId field, but that kinda caused me to rewrite all the ducks stuff for EventAnnouncement.